frisc/harness/frisc.futil

```diff
@@ -0,0 +1,354 @@
+extern "../src/extras/extras.sv" {
```

Comment:
It seems we're committing generated code here which, in general, should be avoided so that the tests can rebuild all the artifacts and make sure things work. I'm assuming this is because of #178 but is there a workaround that we can use in the meantime? Or, if #178 is easy to solve, solving it before merging this?

Reply:
Sure, I'll try to solve #178 first